### DISCUSSION TITLE
How to configure SSO on my own Saml IDP

### DISCUSSION THREAD

**Discussion by habin201291** [gravitational/teleport#35681] [Q&A] [answered]
I'm currently building a simple Saml IDP using Ruby on Rails and the [Saml_idp](https://github.com/saml-idp/saml_idp) gem. Is there a way to configure SSO on my IDP?

Thank!

**Comment by webvictim** [comment, marked as answer]
The Team and Community versions of Teleport do not support the use of SAML or OIDC - just the Enterprise versions.

If you're using Teleport Enterprise, please submit a ticket via our support portal to ask for assistance with this.

For more information on Enterprise versions, see https://goteleport.com/pricing/
